Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/ErrorHandler.java

```diff
@@ -354,9 +359,9 @@ float interpolatedMedian(double[] array) {
         }
     }
 
-    float interpolatedLowerRank(double[] array, double fracRank) {
+    float interpolatedLowerRank(double[] array, double fracRank, double deviation) {
```

Comment:
Can you either rename array to ascendingArray or put the sort inside the method? It will be easier to understand why you use array[0] in lower rank computation and array[len-1] in higher rank computation?

Reply:
Done